CMakeLists.txt

```diff
@@ -203,7 +214,7 @@ if(DEBUG)
 
   # mlpack uses it's own mlpack::backtrace class based on Binary File Descriptor
   # <bfd.h> and linux Dynamic Loader <libdl.h> and more portable version in future
-  if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+  if(CMAKE_SYSTEM_NAME STREQUAL "Linux" AND (NOT CMAKE_CROSSCOMPILING))
```

Comment:
Do you really need to set `AND (NOT CMAKE_CROSSCOMPILING)`?  I think what will happen in most cases is just that Bfd or LibDL can't be found, and then nothing happens.

Reply:
If this is the case, then I am okay with removing it, I have only added it for precaution

Reply:
Give it a shot and let's see what happens. :)  We could also add an option like `SHOW_BACKTRACES` and just set that to `OFF` as the default if we are cross-compiling (and `ON` otherwise), if we do have problems.